packages/dev/core/src/Engines/WebGL/webGLRenderTargetWrapper.ts

```diff
@@ -116,8 +116,8 @@ export class WebGLRenderTargetWrapper extends RenderTargetWrapper {
      * @internal
      * @param renderTarget Destination renderTarget
      */
-    public _shareDepth(renderTarget: WebGLRenderTargetWrapper): void {
-        super._shareDepth(renderTarget);
+    public shareDepth(renderTarget: WebGLRenderTargetWrapper): void {

```

Comment:
does it still stay `@internal`, or is it public API?

Reply:
I forgot to remove the tag, thanks!